types/stripe-v3/index.d.ts

```diff
@@ -856,6 +856,7 @@ declare namespace stripe {
             requestPayerEmail?: boolean | undefined;
             requestPayerPhone?: boolean | undefined;
             requestShipping?: boolean | undefined;
+            disableWallets: any[] | undefined;
```

Comment:
thanks for the contribution @eliasfernandez .

can you please update the field type accordingly to the documentation?
https://stripe.com/docs/js/payment_request/create#stripe_payment_request-options-disableWallets

also if you have time please update the usage in the test class.

Reply:
failing test is related to this change because of stricter undefined checks coming in the ts 4.4 https://github.com/microsoft/TypeScript/issues/13195 - i.e. the new field should be marked as optional explicitly for backward compatibility